System_Information.bat

```diff
@@ -0,0 +1,29 @@
+@echo off
+
+:: Create directory "System Information" if doesn't exist
+echo Create folder: System Information
+IF NOT EXIST "System Information" md "System Information"

```

Comment:
I feel I should mention that it's good practice to instead call `pushd %~dp0` at the top, then at the very end call `popd`. This will restore the current directory at the end.